### ISSUE TITLE
[Android + Chromecast] Bug - Disconnecting from Chromecast clears Music playlist

### ISSUE THREAD

**Issue by BloodyIron** [MediaBrowser/Emby#3424]
Start playing an Album on Emby on my phone.

Connect to Chromecast on WiFi network, starts playing music through speakers. Nice.

Disconnect from Chromecast, and it clears the playlist/album queued up, and of course stops playing any music.

Expected result is for the playlist/queue to not get wiped, and music to transition from the Chromecast device to my Phone and just keep playing at the point I was at.

**Comment by LukePulverenti** [comment]
Hi, this is not true. When you disconnect it asks you if you want to shutdown the chromecast app. Click no.

**Comment by LukePulverenti** [comment]
Yes you're right thanks. We don't support this but we probably should.
